front/src/components/AdminPanel/Panel/CompetitionDetails/EditDetails/handleSubmit.js

```diff
@@ -0,0 +1,7 @@
+export function handleSubmit(history, address, id) {
```

Comment:
"id" parameter is never used. Is it necessary here, then? Also, this function is (apart from aforementioned "id" parameter) identical with one in AdminPanel/NewCompetition/handleSubmit.js file. Maybe it could be merged and placed i.e. in utils/handleSubmit.js? I believe that such handler will be used in other components as well, therefore it seems to be even a better idea to do that.

Reply:
(These handler functions were taken outside to create better tests)

I thought about the same idea, but there's going to be handle API there, so I think it's going to look different for each component, even for NewCompetition end EditCompetition, because when we editing we have to have an id of the competition which details we want to change, when we create a new one  there's no id yet

Reply:
I understand. In that case it can stay.